app/newRoot.css

```diff
@@ -348,3 +351,53 @@ ul {
 .icon-link.focused path {
   stroke: var(--colors-teal-600);
 }
+
+.desktop-only {
+  display: inherit;
+}
+
+.mobile-only {
+  display: none;
+}
+
+@media (max-width: 600px) {
+  .page-body {
+    padding: 0px 6.333vw;
```

Comment:
cool yeah, and let's make sure we use `.page-body` everywhere that uses standard margins, including the footer actually

Reply:
I actually wanted to delete this before pushing :/ I only had it for testing...